packages/error-reporting/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
```

Comment:
You can remove the `$(npm bin)` part of this command and the system-test command.